src/thx/LocalDate.hx

```diff
@@ -511,6 +496,10 @@ Returns true if this date and the `other` date share the same year and month.
 
   inline public function toDate() : Date
     return new Date(year, month - 1, day, 0, 0, 0);
```

Comment:
This is subtle; in Date I take it that months are 0-based? Maybe that should be the same here to avoid confusion?